src/lib/list/selection-list.spec.ts

```diff
@@ -579,3 +658,28 @@ class SelectionListWithTabindexBinding {
   tabIndex: number;
   disabled: boolean;
 }
+
+@Component({template: `
+<mat-selection-list id = "selection-list-5">
```

Comment:
remove the around spaces: `id="selection-list-5"`?

Reply:
I don't know if I hit the format button on my side, but there are spaces around most of the unit test component's id's in this file.

I'll fix them.

Reply:
You're right. There are a lot of strange spaces in this spec file.